rgw/admin/errors_test.go

```diff
@@ -8,11 +8,16 @@ import (
 )
 
 var (
-	fakeGetUserError = []byte(`{"Code":"NoSuchUser","RequestId":"tx0000000000000000005a9-00608957a2-10496-my-store","HostId":"10496-my-store-my-store"}`)
+	fakeGetUserError    = []byte(`{"Code":"NoSuchUser","RequestId":"tx0000000000000000005a9-00608957a2-10496-my-store","HostId":"10496-my-store-my-store"}`)
```

Comment:
Is this change to already existing `fakeGetUser` result of some formatting rules?

Reply:
Yes, it's due to the standard Go formatting (vertically aligning the `=` symbols).